manimlib/default_config.yml

```diff
@@ -5,13 +5,13 @@
 # you are running manim. For 3blue1brown, for instance, mind is
 # here: https://github.com/3b1b/videos/blob/master/custom_config.yml
 
-# Alternatively, you can create it whereever you like, and on running
+# Alternatively, you can create it where ever you like, and on running
```

Comment:
```suggestion
# Alternatively, you can create it wherever you like, and on running
```

See [the Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/wherever)

Reply:
this was marked as resolved although it was not taken into account

Reply:
Thanks!